lookout/style/format/templates/quality_report.md.jinja2

```diff
@@ -29,6 +37,6 @@
 <details>
     <summary>Machine-readable report</summary>
 ```json
-{{ cl_report | tojson }}
+{{ dict(cl_report=cl_report, cl_report_full=cl_report_full, ppcr=ppcr) | tojson }}
```

Comment:
Come on :)

```suggestion
{
  "cl_report": {{ cl_report | tojson }},
  "cl_report_full": {{ cl_report_full | tojson }},
  "ppcr": {{ ppcr | tojson }}
}
```

Reply:
it is more reliable and easy to extend. I prefer to keep as is.

Reply:
It is not more reliable and the same complexity of extension. However, it goes against idiomatic templates and requires an additional filter - `dict` - which is not included for a reason. Please change.